docs/csharp/fundamentals/coding-style/coding-conventions.md

```diff
@@ -322,7 +322,7 @@ In general, use the following format for code samples:
 ### Comment style
 
 - Use single-line comments (`//`) for brief explanations.
-- Avoid multi-line comments (`/* */`) for longer explanations. Comments aren't localized. Instead, longer explanations are in the companion article.
+- Avoid multi-line comments (`/* */`) for longer explanations.<br/>Comments in the code samples aren't localized. Which means explanations embedded in the code will not be translated. So longer, explanatory text should be placed in the companion article, so that it can be localized.
```

Comment:
```suggestion
- Avoid multi-line comments (`/* */`) for longer explanations.<br/>Comments in the code samples aren't localized. That means explanations embedded in the code will not be translated. Longer, explanatory text should be placed in the companion article, so that it can be localized.
```